lib/placeholders/box/readMe.md

```diff
@@ -0,0 +1,30 @@
+## Navbar
```

Comment:
I guess This is added while taking a markdown sample of other components.
Please replace with a relevant title for this component (ex: Box Placeholder).

Reply:
977c4770419b2a5b4289c7e62a8812e8b16cf095